variables.tf

```diff
@@ -308,3 +308,225 @@ variable "check_ec2_imdsv2" {
   type        = bool
   default     = false
 }
+
+variable "check_access_keys_rotated" {
+  description = "Enable access-keys-rotated rule"
+  type        = bool
+  default     = true
+}
+
+variable "access_key_max_age" {
+  description = "Maximum number of days without rotation."
+  type        = number
+  default     = 90
+}
+
+variable "check_cmk_backing_key_rotated" {
+  description = "Enable cmk_backing_key_rotation_enabled rule"
```

Comment:
What is `cmk backing key`?

Reply:
This is for Customer Managed Keys in Key Management Service. (KMS) [cmk-backing-key-rotation-enabled](https://docs.aws.amazon.com/config/latest/developerguide/cmk-backing-key-rotation-enabled.html) Checks that Customer Managed Key rotation is enabled.